src/api/Alert/index.ts

```diff
@@ -329,11 +330,32 @@ export class AlertAPI extends Request {
    */
   public getFullCaseList(this: Request<IFullCaseListResponse>, params: IGetFullCaseListParams) {
     const sortByParamToString = JSON.stringify(params.sortBy);
+    const alertIdsFilter = JSON.stringify(params.alertIdsFilter);
```

Comment:
I thought we aligned on using comma-separated values for arrays passed as query-string params, was there a difficulty implementing it?